static/app/views/replays/detail/tagPanel/index.tsx

```diff
@@ -68,25 +70,34 @@ function TagPanel() {
     <FluidHeight>
       <TagFilters tags={tags} {...filterProps} />
       <TabItemContainer>
-        <FluidPanel>
-          {filteredTags.length ? (
-            <KeyValueTable noMargin>
-              {filteredTags.map(([key, values]) => (
-                <ReplayTagsTableRow
-                  key={key}
-                  name={key}
-                  values={values}
-                  generateUrl={generateUrl}
-                />
-              ))}
-            </KeyValueTable>
-          ) : (
-            <EmptyMessage>{t('No tags for this replay were found.')}</EmptyMessage>
-          )}
-        </FluidPanel>
+        <StyledPanel>
+          <FluidPanel>
```

Comment:
this `<FluidPanel>` doesn't seem like its doing anything important

Reply:
<FluidPanel> helps with the row heights when doing searches

Reply:
ooh, ok 👍